doc_src/getting_started.md

```diff
@@ -35,5 +35,5 @@
     - Take a look around and play around with it.
     - Check out documentations and see what you can do with antikythera and gears.
         - [Guide for instance administrators](./instance_administrators/) (TBD)
```

Comment:
Outside this PR, links to non-existent pages look unnatural.
Would you like to temporarily disable the link?
```suggestion
         - Guide for instance administrators[](./instance_administrators/) (TBD)
```

Reply:
Thank you for your suggestion.
I applied.
2bbe9751e3d6bda1c815d990c6fe7a555084ba21